microcosm_fastapi/conventions/crud_adapter.py

```diff
@@ -62,6 +62,8 @@ def search(self, offset: int, limit: int) -> SearchSchema(PizzaSchema):
         payload = dict(
             items=items,
             count=count,
+            offset=offset,
+            limit=limit,
```

Comment:
This change isn't essential as you can grab these from the controller inputs but might be easier to have offset and limit come from the crud_adapter to save having to put them into every application controller we need them in.